SConstruct

```diff
@@ -7,6 +7,16 @@ import tarfile
 import zipfile
 from SCons.Script.SConscript import SConsEnvironment
 
+# Add a check_output function if we are running under Python 2.6...
```

Comment:
This comment no longer says what I requested. And please don't end a comment with `...`, it makes it look like you wanted to say something else but decided to keep the reader in suspense.